doc/api/errors.md

```diff
@@ -589,6 +585,30 @@ program. For a comprehensive list, see the [`errno`(3) man page][].
   encountered by [`http`][] or [`net`][] — often a sign that a `socket.end()`
   was not properly called.
 
+## OpenSSL Errors
+
+Errors originating in `crypto` or `tls` are of class `Error`, and in addition to
+the standard `.code` and `.message` properties, may have some additional
+OpenSSL-specific properties.
+
+### error.opensslErrorStack
```

Comment:
Is it worth to sort these subsections alphabetically?

Reply:
partially sorted, I can full-sort if you'd like, though its a bit odd because it puts the error stack in the middle of the other string properties

Reply:
I am OK with whatever you prefer here)